ncs/Kconfig

```diff
@@ -4,6 +4,27 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+# The following nine KConfigs are maintained for backward compatibility with the sdk-nrf-next repository.
+# They are slated for removal once sdk-nrf-next is no longer in use.
+config SOC_NRF54H20_ENGA_CPUAPP
+    bool
+config SOC_NRF54H20_CPUAPP
```

Comment:
This symbol should be defined in NCS

Reply:
It seems that CONFIG_SOC_NRF54H20_ENGA_CPUAPP is defined instead.

Reply:
Removed since CONFIG_SOC_NRF54H20_ENGA_CPUAPP  not available anymore